src/sync/index.js

```diff
@@ -429,20 +456,67 @@ async function updateCentralizedOraclesPassedResultSetEndBlock(currentBlockChain
       {$set: {status:'OPENRESULTSET'}}, {multi: true});
     console.log('Updated COracles Passed ResultSetEndBlock');
   }catch(err){
-    console.error(`Error: updateCentralizedOraclesPassedResultSetEndBlock ${err.message}`);
+    console.error(`ERROR: updateCentralizedOraclesPassedResultSetEndBlock ${err.message}`);
   }
 }
 
+async function updateOracleInfo(db, oraclesNeedInfoUpdate) {
+  _.forEach(oraclesNeedInfoUpdate, async (oracleAddress, index) => {
+    let oracle;
+    try {
+      oracle = await db.Oracles.findOne({ 
+        address: oracleAddress 
```

Comment:
use `_id`, it is an indexed primary key

Reply:
The `_id` field of the TopicEvent isn't in the Oracle objects.